plugin_scores_cache.yml

```diff
@@ -9974,14 +9974,14 @@ fastlane-plugin-teams_message_card:
   :github_issues: 0
   :github_forks: 0
   :github_contributors: 0
-fastlane-plugin-microsft_teams_message_card:
+fastlane-plugin-microsoft_teams_message_card:
   :initial_commit: 2020-10-02
   :age_in_days: 10
   :readme_score: 50
   :tests: 1
   :actions:
   - !ruby/object:FastlaneCore::Helper::PluginScoresHelper::FastlanePluginAction
-    name: microsft_teams_message_card
```

Comment:
This typo seems correct, unfortunately... 😞  
https://rubygems.org/gems/fastlane-plugin-microsft_teams_message_card

Reply:
thanks, reverted 🔨